src/modules/events.js

```diff
@@ -1,5 +1,60 @@
-const taskCompleted = () => {
+/* eslint-disable linebreak-style */
+import { getTasks } from './functions.js';
 
+export const taskAdd = (event, list) => {
```

Comment:
- [ ] Kindly make sure the property index is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). Take a look at the screenshot below, you will see there is a 5th task is added but its index is set to 4. Kindly make changes,
![Screenshot (200)](https://user-images.githubusercontent.com/67087664/155373910-32dda078-7b4b-4c4f-a8c9-00d27f313531.png)
 please.

Reply:
Hello Elyor.
Thanks for reviewing my code.
Is it possible to meet on zoom call now?

Reply:
this is a demonstration for indexes point:
[indexes explanation video](https://www.loom.com/share/13bac3953b644f638b4f7d8771c3a484)

Reply:
Hey @saied2035, I'm your new reviewer. Saw your video. I contacted you on Slack.

Reply:
@Meltrust 
I responded on slack.